app/src/main/java/com/example/pokedex/pokemonDetails/pagerFragments/MovesAdapter.kt

```diff
@@ -0,0 +1,36 @@
+package com.example.pokedex.pokemonDetails.pagerFragments
+
+import android.view.LayoutInflater
+import android.view.ViewGroup
+import android.widget.TextView
+import androidx.recyclerview.widget.RecyclerView
+import com.example.pokedex.databinding.FormsRecyclerItemBinding
+import com.example.pokedex.pokemonDetails.detailsModel.Move
+
+class MovesAdapter(private val movesList: List<Move>) :
+    RecyclerView.Adapter<MovesAdapter.MovesViewHolder>() {
+
+    inner class MovesViewHolder(binding: FormsRecyclerItemBinding) :
+        RecyclerView.ViewHolder(binding.root) {
+
+        private val moveName: TextView = binding.abilityName
```

Comment:
this is not necessary, you can use the binding directly

Reply:
Done